src/ReachSets/ContinuousPost/BFFPS19/reach.jl

```diff
@@ -42,7 +42,7 @@ Interface to reachability algorithms for an LTI system.
 
 ### Output
 
-A sequence of [`ReachSet`](@ref)s.
+A sequence of [`AbstractReachSet`](@ref)s.
```

Comment:
I don't quite understand -- wound't this be a `SparseReachSet`?

Reply:
True!

Reply:
Ah, actually there is a bug. I wanted to have a mixture because usually we have low- and high-dimensional sets there. But now I only create low-dimensional `SparseReachSet`s all the time, which is clearly wrong.

Reply:
We agreed that `BFFPS19` should always return a `SparseReachSet`.